jsapp/js/components/submissions/submissionUtils.tsx

```diff
@@ -84,10 +84,6 @@ export class DisplayGroup {
       this.xpathNodes = xpathNodes
     }
   }
-
-  addChild(child: DisplayResponse | DisplayGroup) {
```

Comment:
I think removing this function and instead using `children.push` directly makes it technically less safe (because there's no restriction on what can be pushed now)? I'm sure co-pilot would've caught it that is really true, seeing as `parentGroup` is typed.

Reply:
If I add `parentGroup.children.push('a')` somewhere in the function, TS will display an error :)

Reply:
I think it's safe enough to use